internal/pkg/cosign/rekor/util_test.go

```diff
@@ -0,0 +1,155 @@
+}
+
+func TestRekorbytes(t *testing.T) {
+	key, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+	if err != nil {
+		t.Fatalf("could not generate key: %v", err)
+	}
+	testPub := &key.PublicKey
+	testPubPEM, err := cryptoutils.MarshalPublicKeyToPEM(testPub)
+	if err != nil {
+		t.Fatalf("could not marshall public key to PEM: %v", err)
+	}
+
+	testCases := []struct {
+		desc string
+
+		cert *x509.Certificate
+		pub  crypto.PublicKey
+
+		expected []byte
+	}{
+		{
```

Comment:
Use `{{` and `}, {` to reduce the bloat on these.

Reply:
done